src/components/SettingsLabel.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useColorScheme } from '$app/common/colors';
+
+interface Props {
+  label: string;
+  helpLabel?: string | null;
+  required?: boolean;
+}
+
+export function SettingsLabel(props: Props) {
```

Comment:
@beganovich Sure, I've explained my reasons for using this approach instead of custom hooks. Please refer to the comment above and share your thoughts.